server/src/main/java/io/crate/role/RoleManagerService.java

```diff
@@ -74,8 +75,9 @@ public RoleManagerService(NodeClient client,
     public CompletableFuture<Long> createRole(String roleName,
                                               boolean isUser,
                                               @Nullable SecureHash hashedPw,
-                                              @Nullable JwtProperties jwtProperties) {
-        CreateRoleRequest request = new CreateRoleRequest(roleName, isUser, hashedPw, jwtProperties);
+                                              @Nullable JwtProperties jwtProperties,
+                                              @Nullable Map<String, Object> sessionSettings) {
+        CreateRoleRequest request = new CreateRoleRequest(roleName, isUser, hashedPw, jwtProperties, sessionSettings);
```

Comment:
I think the changes here should be reverted?

Reply:
yes, sorry, I forgot to revert the request stuff.